runtimes/polimec/src/lib.rs

```diff
@@ -1195,17 +1221,27 @@ impl_runtime_apis! {
 			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
 			use xcm::latest::prelude::*;
 			use pallet_xcm::benchmarking::Pallet as PalletXcmExtrinsiscsBenchmark;
-			/// TODO: Update these benchmarks once we enable PLMC Teleportation
+			/// TODO: Update these janky implementations once we upgrade pallet_xcm. The jankyness is due to badly written pallet-xcm code
 			impl pallet_xcm::benchmarking::Config for Runtime {
 				fn reachable_dest() -> Option<MultiLocation> {
+					PolkadotXcm::force_xcm_version(
+						RuntimeOrigin::root(),
+						Box::new(crate::xcm_config::AssetHubLocation::get()),
+						xcm::prelude::XCM_VERSION
+					).unwrap();
 					Some(crate::xcm_config::AssetHubLocation::get())
 				}
 
 				fn reserve_transferable_asset_and_dest() -> Option<(MultiAsset, MultiLocation)> {
+					PolkadotXcm::force_xcm_version(
+						RuntimeOrigin::root(),
+						Box::new(crate::xcm_config::AssetHubLocation::get()),
+						xcm::prelude::XCM_VERSION
+					).unwrap();
 					Some((
 						MultiAsset {
-							fun: Fungible(EXISTENTIAL_DEPOSIT),
-							id: Concrete(Parent.into())
+							fun: Fungible(ExistentialDeposit::get()),
+							id: Concrete(Here.into())
```

Comment:
Why should this be Here? As Here describes PLMC in this context, right? Instead of Dot?

Reply:
Benchmarks use Balances pallet to fund the accounts before the reserve transfer, so we only have the choice of specifying our native asset which was one of the bugs solved in this PR

Reply:
Hmmm interesting that it works with Here, as we should not be able to reserve_based transfer PLMC.